### PR TITLE
Fixed diamond shapes datapoint to align with the time they occur in

### PR DESCRIPTION
**Pull request by pranav300**
**What is the purpose of this pull request? (Check any that's applicable)**

-   [ ] Feature
-   [X] Bug fix
-   [ ] Dependency updates
-   [ ] Documentation updates
-   [ ] Build related changes
-   [ ] Changes an existing functionality
-   [ ] Other, please explain:

**Does this introduce a breaking change?**

-   [ ] Yes
-   [X] No

**What changes did you make? (Give an overview)**
Update scale unit for shape

**Add any screenshots**
Before
Screenshot 2020-04-27 at 9 53 48 AM

After
Screenshot 2020-04-27 at 9 58 07 AM

### CODE REVIEW

**Comment by mraplumb** [comment]
@pranav300 is this a fix for https://github.com/cerner/carbon-graphs/issues/177 or another issue?

**Comment by pranav300** [comment]
Oops, My fault forgot mention that in commit message and description as well.

>@pranav300 is this a fix for #177 or another issue?

Yes, I Will mention it in future commits.

**Comment by jeremyfuksa** [comment]
I've pulled and built this branch. Is there a test I can look at to verify this?

**Comment by pranav300** [comment]
>I've pulled and built this branch. Is there a test I can look at to verify this?

Currently, there are no tests to verify these changes. 
You can add this `shape: Carbon.helpers.SHAPES.DARK.DIAMOND,` before [line 52](https://github.com/cerner/carbon-graphs/blob/master/dev/examples/data.js#L52), And verify the changes on Line Simple example.

File: src/main/js/core/Shape/shapeDefinitions.js [hunk 1]
```diff
@@ -86,8 +86,8 @@ export const TEAR_ALT = {
  * @type {{path: {d: string}, options: {x: number, y: number, scale: number}}}
  */
 export const DIAMOND = {
-    path: { d: "M24 0l14 24-14 24-14-24L24 0z" },
-    options: { x: -8, y: -8, scale: 0.25 }
+    path: { d: "M24 0l12 24-12 24-12-24L24 0z" },
+    options: { x: -8, y: -8, scale: 0.34 }
```
Comment (Veera81) [inline src/main/js/core/Shape/shapeDefinitions.js:90]: Currently, there are no tests for these changes.

So, What would be the testing consideration for these changes from the development perspective before we release this bug?

Comment (pranav300) [reply src/main/js/core/Shape/shapeDefinitions.js:90]: I think there is a plan to add wdio testing to carbon-graphs. But I don't know when we are going to implement it.

cc / @ShettyAkarsh

Comment (ShettyAkarsh) [reply src/main/js/core/Shape/shapeDefinitions.js:90]: Yes, @pranav300  can you add the screenshot of before and after changes ?

**Comment by AshishMotanamGurunadham** [comment]
Are we good with merging this PR?

**Pull request merged**